### PR TITLE
Add Brakeman

### PR DESCRIPTION
**Pull request by jamesrose**
> Brakeman is a static analysis tool which checks Ruby on Rails applications for security vulnerabilities.

https://github.com/presidentbeef/brakeman

### CODE REVIEW

File: .github/workflows/brakeman-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+name: Brakeman Scan
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ develop ]
+  schedule:
+    - cron: '27 13 * * 2'
```
Comment (ethanmills) [inline .github/workflows/brakeman-analysis.yml:13]: Alright... At 13:27 on Tuesday it is

**Review by ethanmills — APPROVED** [review]

**Comment by ethanmills** [comment]
Looks like it's found some stuff https://github.com/DFE-Digital/early-careers-framework/security/code-scanning?query=pr%3A1120+tool%3ABrakeman

File: .github/workflows/brakeman-analysis.yml (hunk 1 above)
Comment (jamesrose) [reply .github/workflows/brakeman-analysis.yml]: Hah.. updated to every day at 12.

**Comment by ethanmills** [comment]
I'm not a fan of brakeman's version parsing 
image
https://github.com/DFE-Digital/early-careers-framework/security/code-scanning/3?query=ref%3Arefs%2Fpull%2F1120%2Fmerge

**Comment by jamesrose** [comment]
@ethanmills What's the issue there?

**Comment by ethanmills** [comment]
This is fixed in Brakeman 5.0.2

**Pull request merged**